CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [2.61.0]
```

Comment:
Remember that here you need to put a reference for the subsequent version, so if the version in your PR is 2.61.0, the job will bump it to 2.62.0, want that is what you should be using here.

Reply:
Done. thanks for feedback.